src/Riok.Mapperly/Emit/SyntaxFactoryHelper.cs

```diff
@@ -323,7 +326,7 @@ public static SeparatedSyntaxList<T> CommaSeparatedList<T>(IEnumerable<T> nodes,
         => SeparatedList<T>(JoinByComma(nodes, insertTrailingComma));
 
     public static IdentifierNameSyntax NonNullableIdentifier(ITypeSymbol t)
-        => IdentifierName(t.NonNullable().ToDisplayString());
+        => FullyQualifiedNonNullableIdentifier(t);
```

Comment:
Couldn't this be simplified to `FullyQualifiedIdentifierName(t.NonNullable())`?

Reply:
Ok I'll update it

Reply:
Thanks for feedback. I updated code according to your feedback.

Please let me know if there are any other additional changes or improvements you would like me to make. Thank you!

Reply:
updated